src/Orders.sol

```diff
@@ -5,103 +5,110 @@ import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 /// @notice Contract capable of processing fulfillment of intent-based Orders.
 abstract contract OrderDestination {
-    /// @notice Emitted when an swap order is fulfilled by the Builder.
-    /// @param originChainId - The chainId on which the swap order was submitted.
+    /// @notice Emitted when an Order output is fulfilled.
     /// @param token - The address of the token transferred to the recipient. address(0) corresponds to native Ether.
-    /// @param recipient - The recipient of the token.
     /// @param amount - The amount of the token transferred to the recipient.
-    event SwapFulfilled(
-        uint256 indexed originChainId, address indexed token, address indexed recipient, uint256 amount
-    );
+    /// @param recipient - The recipient of the token.
+    event OrderFulfilled(address indexed recipient, address indexed token, uint256 amount);
 
     /// @notice Fulfill a rollup Swap order.
-    ///         The user calls `swap` on a rollup; the Builder calls `fulfillSwap` on the target chain.
-    /// @custom:emits SwapFulfilled
-    /// @param originChainId - The chainId of the rollup on which `swap` was called.
+    ///         The user calls `initiate` on a rollup; the Builder calls `fill` on the target chain.
+    /// @custom:emits OrderFulfilled
+    /// @param recipient - The recipient of the token.
     /// @param token - The address of the token to be transferred to the recipient.
     ///                address(0) corresponds to native Ether.
-    /// @param recipient - The recipient of the token.
     /// @param amount - The amount of the token to be transferred to the recipient.
-    function fulfillSwap(uint256 originChainId, address token, address recipient, uint256 amount) external payable {
+    function fill(address recipient, address token, uint256 amount) external payable {
```

Comment:
why not make this a struct too?

Reply:
we explored doing this earlier, and at the time we decided it was cleaner code for Fillers to call this function multiple times - once per Output - rather than arrays of structs.

passing arrays when initiating the order adds new functionality (tying more complex inputs/outputs to an Order than one-for-one) but passing arrays here doesn't add new functionality 

all that said, it probably makes sense to align style between the two contracts now that we've shifted the initiate side...

Reply:
I considered doing an array of Outputs here to match style, but I didn't like that the Output struct contains the `targetChainId`, which is self-evident on the target chain itself; passing extra info as calldata && emitting the same extra info adds gas costs without benefits in visibility

Reply:
> all that said, it probably makes sense to align style between the two contracts now that we've shifted the initiate side...

this is what i'm more concerned about. could easily have both single and list?

Reply:
explored this in a separate branch: https://github.com/init4tech/zenith/pull/53

honestly think what's in this branch kinda makes more sense.. 

maybe at this point let's wrap up this PR and then decide on the other branch as a fast follow?